swagger.json

```diff
@@ -5850,6 +6001,72 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "User": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "The id of the User record.",
+                    "example": 123
+                },
+                "name" : {
+                    "type": "string",
+                    "description": "The first and last name of the user.",
+                    "example": "Jane Doe"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "The email address of this user.",
+                    "example": "user@company.com"
+                },
+                "organizationRole" : {
+                    "type": "string",
+                    "description": "The name of the role the user is assigned to for the organization. This will be blank for users that only have access to specific tags.",
+                    "example": "Read-Only Admin"
+                },
+                "tagRoles" : {
+                    "type": "array",
+                    "description": "The specific tags this user has access to. This will be blank for users that have full access to the organization.",
+                    "items": {
+                        "$ref": "#/definitions/UserTagRole"
+                    }
+                },
+                "authType": {
+                    "type": "string",
+                    "enum": [
+                        "basic",
+                        "saml"
+                    ],
+                    "description": "The authentication type the user uses to authenticate. To use SAML your organization must have a configured SAML integration."
+                }
+            }
+        },
+        "UserTagRole": {
+            "type": "object",
+            "properties": {
+                "tag": {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "The ID of this tag.",
+                            "example": 12345
+                        },
+                        "name": {
```

Comment:
Should we return a full tag object here? (Including `parentTagId`, etc)

Reply:
I thought about doing that, but I am not sure what a consumer would do with that information. Do you have a preference?

Reply:
No preference, I'm primarily concerned that this field is future compatible with the tag object in case we decide to include additional tag fields (e.g. external_ids).

Reply:
Good point, I will add `parentTagId`.